tmuxp/cli.py

```diff
@@ -924,3 +925,11 @@ def command_convert(config):
                 buf.write(newconfig)
                 buf.close()
                 print('New config saved to <%s>.' % newfile)
+
+
+@cli.command(name='ls', short_help='List configured sessions in $HOME/.tmuxp dir.')
+def command_ls():
+    tmuxp_dir = Path.home() / '.tmuxp'
+    if tmuxp_dir.exists() and tmuxp_dir.is_dir():
+        for f in tmuxp_dir.iterdir():
```

Comment:
This is my `.tmuxp` dir: https://github.com/tony/tmuxp-config

When I tried `tmuxp ls`:

```
js-frontend
cpython
LICENSE
eduflow
scripts
braintree
README
_bootstrap_util
python
notifly
peergrade
saltstack
devops
pyterm
freebsd
.gitignore
.git
```

We may want to filter out files that aren't valid extensions (`['.yaml', '.yml', '.json']`, see [here](https://github.com/tmux-python/tmuxp/blob/7649b245fbfcc93ab447a074a4d24f4522493c6b/tmuxp/cli.py#L330))